### PR TITLE
[ARTEMIS-1761] Log one warning if cluster could not be formed because…

### PR DESCRIPTION
**Pull request by rstancel**
… it's not possible to connect to other cluster node

Jira: https://issues.apache.org/jira/browse/ARTEMIS-1761

### CODE REVIEW

**Comment by clebertsuconic** [comment]
I amended the commit title... too big.. 

I am not strict with rules.. I just try to be reasonable..  sometimes even I bend them.. about number of characters per line on commit descriptions and the first line

but your commit line was huge.. impossible to read it on git logs... 


I use git log all the time.. so this would be an issue.. just to let you know in the future... 


Thanks for the commit BTW! :)

File: artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java [hunk 1]
```diff
@@ -927,9 +927,7 @@ protected void connect() {
                scheduleRetryConnectFixedTimeout(this.retryInterval);
                return;
             } else {
-               if (logger.isDebugEnabled()) {
-                  logger.debug("Bridge " + this + " is unable to connect to destination. Retrying", e);
-               }
+               logger.warn("Bridge " + this + " is unable to connect to destination. Retrying...");
```
Comment (michaelandrepearce) [inline artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java]: Can we use the logger class's for this, as its no longer a debug message, should have proper code etc. etc.

Comment (rstancel) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java]: @michaelandrepearce I have used the same message as in another if-else condition. is it ok?

Comment (michaelandrepearce) [reply artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java]: Not really it should have its own, it’s a different issue, and so should have its own dedicated log error code and log output.

**Comment by rstancel** [comment]
@michaelandrepearce Done. Can you please check? Thanks

**Comment by clebertsuconic** [comment]
i would squash these three commits into one... but we can do that at time of merge..

**Comment by rstancel** [comment]
@clebertsuconic yeap, I will do that once it will be approved

**Comment by clebertsuconic** [comment]
@rstancel I already rebased and merged it

**Comment by clebertsuconic** [comment]
squashed it with an intermediate branch.

**Pull request closed without merging**